app/models/question_answer_vote.rb

```diff
@@ -31,7 +31,7 @@ def self.reverse_direction(direction)
   def ensure_valid_post
     if !post.qa_enabled
       errors.add(:base, I18n.t("post.qa.errors.qa_not_enabled"))
-    elsif post.post_number == 1 || post.reply_to_post_number.present?
+    elsif post.reply_to_post_number.present?
```

Comment:
Thinking out loud: `post.reply_to_post_number.present?` means "this post is an answer"?

Reply:
A post can be in reply to another post and we use this column in core to indicate that.